packaging/README.md

```diff
@@ -0,0 +1,52 @@
+# Releasing pg-backup-api
+
+## Increment the version
+
+Update the version string in [pg_backup_api/version.txt](../pg_backup_api/version.txt).
+Use [semantic versioning](https://semver.org/) when determining the new version.
+
+If you are publishing new packages for a version of pg-backup-api which has already been published then the pg-backup-api version should remain the same.
+
+## Set the package release version
+
+### New pg-backup-api releases
+
+Check that the `RELEASE_VERSION` in [.github/workflows/publish-to-cloudsmith.yml](../.github/workflows/publish-to-cloudsmith.yml) is set to `1`.
+
+### Re-publishing existing pg-backup-api releases
+
+Increment the `RELEASE_VERSION` in [.github/workflows/publish-to-cloudsmith.yml](../.github/workflows/publish-to-cloudsmith.yml).
+
+## Update the release notes
+
+* Add release notes to [pg_backup_api/news.yml](../pg_backup_api/news.yml).
+* Make sure you include your name and email in the `author` field as this will be included in the rpm and deb changelogs.
+* If you are re-publishing an existing pg-backup-api release then include the `iteration` field and set this to the value of `RELEASE_VERSION` used in the above step.
+* Generate the changelogs by running `./changelogs.py`.
+* Move the changelogs into place:
+  * `mv templates/deb.changelog changelogs`
+  * `mv templates/rpm.changelog changelogs`
```

Comment:
I'm probably missing something, but wouldn't following this sequence exactly just clobber the deb changelogs file with the rpm one?

Reply:
If the last argument passed to `mv` is an existing directory then it will move the files referenced by the preceding arguments into the directory, so in this case both `deb.changelog` and `rpm.changelog` get moved into `changelogs`. We could simplify it further and just say `mv templates/deb.changelog templates/rpm.changelog changelogs`.

Reply:
Oh, the thing I missed is that `changelogs` is a directory, not a file. Got it! Thanks.